src/test/run-make-fulldeps/coverage/uses_crate.rs

```diff
@@ -0,0 +1,10 @@
+#![allow(unused_assignments, unused_variables)]
+
+mod used_crate;
```

Comment:
This is a different module in the same crate. To test two crates, you need two rustc invocations.

Reply:
Done